news/feature-3474.md

```diff
@@ -0,0 +1,5 @@
+logwriter based destinations: new truncate_size option introduced to truncate an output message to a specified max size. default value is -1 (disabled).
```

Comment:
I think we should just list those destinations, `logwriter` does not say much for an end-user:

`file, network, program destinations: new truncate-size() option`. 

Can we use dashes in option names, please?

Reply:
Sure.